lib/web_ui/lib/src/engine/semantics/text_field.dart

```diff
@@ -460,6 +460,6 @@ class TextField extends RoleManager {
     if (!isIosSafari) {
       editableElement?.remove();
     }
-    SemanticsTextEditingStrategy.instance.deactivate(this);
+    SemanticsTextEditingStrategy._instance?.deactivate(this);
```

Comment:
Is it possible to add an engine test that simulates the situation where dispose is called before the strategy is initialized?